pkg/cmd/source.go

```diff
@@ -90,7 +90,7 @@ func (o *SourceClientOptions) Completed() *SourceClientConfig {
 
 func getSourceScheme() *runtime.Scheme {
 	scheme := runtime.NewScheme()
-	utilruntime.Must(corev1.AddToScheme(scheme))
+	utilruntime.Must(kubernetesscheme.AddToScheme(scheme))
```

Comment:
FYI: We have seen issues with event reporting for `Lease` objects now that GRM uses `configmapsleases` as default leader election resource lock. We decided that we can simply add all Kubernetes APIs to our source scheme here.